common/hooks/pre-pkg/99-pkglint.sh

```diff
@@ -21,13 +21,40 @@ hook() {
 		fi
 	done
 	
-	for f in sys dev home root run var/run tmp usr/local destdir; do
+	for f in var/run usr/local; do
 		if [ -d ${PKGDESTDIR}/${f} ]; then
 			msg_red "${pkgver}: /${f} directory is not allowed, remove it!\n"
 			error=1
 		fi
 	done
 
+	for f in "$PKGDESTDIR"/*; do
+		f="${f##*/}"
+		case "$f" in
+		'*')	# The filename is exactly '*'
+			if [ -f "${PKGDESTDIR}/*" ]; then
+				msg_red "${pkgver}: File /${f} is not allowed\n"
+				error=1
+			elif [ "$build_style" != "meta" ]; then
```

Comment:
It passes package not being `meta` containing just INSTALL.msg with this change.

Reply:
I think I'll just drop this `'*'` case.
`xbps-src` will always create `INSTALL` file, after all?

Or is it better to keep it, add a variable `nonmetafile`, set it to 1 in `lib|...`?
using find later for checking empty package is not the best solution, I think